acceptance/tests/pxp_module_bolt_command/run_command.rb

```diff
@@ -25,16 +25,24 @@
     end
   end # test step
 
-  step 'Responses that create large output which exceedes max-message-size error' do
-    agents.each do |agent|
-      # Escaping quotes for powershell turns out to be more confusing than just
-      # getting away from quotes by coercing an integer to a string
-      win_cmd = '$x = 1; $y = $x.ToString() * 65 * 1012 * 1012; Write-Output($y)'
-      nix_cmd = '/opt/puppetlabs/puppet/bin/ruby -e "puts \'a\'* 65 * 1012 * 1012"'
-      cmd = agent.platform =~ /windows/ ? win_cmd : nix_cmd
-      run_errored_command(master, [agent], cmd) do |stdout|
-        assert_match(/exceeded max-message-size/, stdout)
+  # This test takes a bunch of resources to process correctly (and
+  # not produce false positive failures). Platforms like MacOS, AIX
+  # and older windows have a hard time reliably running the test
+  # so we just confine the platforms to the standard Linuxes
+  if agent.platform =~ /(el|ubuntu)/
```

Comment:
Not sure i understand how this restricts? Isnt `agents` *all* the agents in the test? 

Would the logic look more like: collect a subset of `agents` then iterate over those for this specific test?

Reply:
Ah, good catch.

Reply:
Thinking something similar to https://github.com/puppetlabs/pxp-agent/blob/master/acceptance/tests/proxy_connection.rb#L20